templates/base/graphQl/ApolloClient.js

```diff
@@ -2,17 +2,18 @@ import ApolloClient, { HttpLink, InMemoryCache } from 'apollo-boost';
 import { ApolloLink } from 'apollo-link';
 import { onError } from 'apollo-link-error';
 import { graphQLApiUri } from '../config/endpoints';
+import Logger from '../src/utils/Logger';
 
 const cache = new InMemoryCache();
 
 const errorLink = onError(({ graphQLErrors, networkError }) => {
   if (graphQLErrors)
     graphQLErrors.forEach(({ message, locations, path }) =>
-      console.log(
-        `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`
+      Logger.error(
+        { message: `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}` }
       )
     );
-  if (networkError) console.log(`[Network error]: ${networkError}`);
+  if (networkError) Logger.error({ message: `[Network error]: ${networkError}` });
```

Comment:
@all9planets For error the object **error** and **service** in log data becomes mandatory.

Reply:
done